config/config.go

```diff
@@ -104,32 +101,32 @@ type Config struct {
-	Status                     Status             `toml:"status" json:"status"`
-	Performance                Performance        `toml:"performance" json:"performance"`
-	PreparedPlanCache          PreparedPlanCache  `toml:"prepared-plan-cache" json:"prepared-plan-cache"`
-	OpenTracing                OpenTracing        `toml:"opentracing" json:"opentracing"`
-	ProxyProtocol              ProxyProtocol      `toml:"proxy-protocol" json:"proxy-protocol"`
-	PDClient                   PDClient           `toml:"pd-client" json:"pd-client"`
-	TiKVClient                 tikvcfg.TiKVClient `toml:"tikv-client" json:"tikv-client"`
-	Binlog                     Binlog             `toml:"binlog" json:"binlog"`
-	CompatibleKillQuery        bool               `toml:"compatible-kill-query" json:"compatible-kill-query"`
-	Plugin                     Plugin             `toml:"plugin" json:"plugin"`
-	PessimisticTxn             PessimisticTxn     `toml:"pessimistic-txn" json:"pessimistic-txn"`
-	CheckMb4ValueInUTF8        bool               `toml:"check-mb4-value-in-utf8" json:"check-mb4-value-in-utf8"`
-	MaxIndexLength             int                `toml:"max-index-length" json:"max-index-length"`
-	IndexLimit                 int                `toml:"index-limit" json:"index-limit"`
-	TableColumnCountLimit      uint32             `toml:"table-column-count-limit" json:"table-column-count-limit"`
-	GracefulWaitBeforeShutdown int                `toml:"graceful-wait-before-shutdown" json:"graceful-wait-before-shutdown"`
+	LowerCaseTableNames        int                    `toml:"lower-case-table-names" json:"lower-case-table-names"`
+	ServerVersion              string                 `toml:"server-version" json:"server-version"`
+	Log                        Log                    `toml:"log" json:"log"`
+	Security                   Security               `toml:"security" json:"security"`
+	Status                     Status                 `toml:"status" json:"status"`
+	Performance                Performance            `toml:"performance" json:"performance"`
+	PreparedPlanCache          PreparedPlanCache      `toml:"prepared-plan-cache" json:"prepared-plan-cache"`
+	OpenTracing                OpenTracing            `toml:"opentracing" json:"opentracing"`
+	ProxyProtocol              ProxyProtocol          `toml:"proxy-protocol" json:"proxy-protocol"`
+	PDClient                   tikvcfg.PDClient       `toml:"pd-client" json:"pd-client"`
+	TiKVClient                 tikvcfg.TiKVClient     `toml:"tikv-client" json:"tikv-client"`
+	Binlog                     Binlog                 `toml:"binlog" json:"binlog"`
+	CompatibleKillQuery        bool                   `toml:"compatible-kill-query" json:"compatible-kill-query"`
+	Plugin                     Plugin                 `toml:"plugin" json:"plugin"`
+	PessimisticTxn             tikvcfg.PessimisticTxn `toml:"pessimistic-txn" json:"pessimistic-txn"`
```

Comment:
It's not a part of TiKV.

Reply:
except for `PerssimisticTxn`?

Reply:
However, it seems it only used in tikv now?

Reply:
done